src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/parsed/RabbitParsedBatchSubscriber.java

```diff
@@ -33,12 +33,21 @@
 
 public class RabbitParsedBatchSubscriber extends AbstractRabbitBatchSubscriber<Message, MessageBatch> {
 
-    private static final Counter INCOMING_PARSED_MSG_BATCH_QUANTITY = Counter.build("th2_mq_incoming_parsed_msg_batch_quantity", "Quantity of incoming parsed message batches").register();
-    private static final Counter INCOMING_PARSED_MSG_QUANTITY = Counter.build("th2_mq_incoming_parsed_msg_quantity", "Quantity of incoming parsed messages").register();
+    private static final Counter INCOMING_PARSED_MSG_BATCH_QUANTITY = Counter.build()
+            .name("th2_mq_incoming_parsed_msg_batch_quantity")
+            .labelNames("session alias")
```

Comment:
Please replace space to underscore and extract session_alias to const into the CommonMetrics.kt